app/index.js

```diff
@@ -43,12 +44,25 @@ bot.on('message', (msg) => {
                 return daily(body, bot, chatId)
             }
 
-            const pattern = /daily (\d+)/
-            if (pattern.test(userMessage)) {
-                const n = pattern.exec(userMessage)[1]
+            const dailyPattern = /daily (\d+)/
+            if (dailyPattern.test(userMessage)) {
+                const n = dailyPattern.exec(userMessage)[1]
                 return daily(body, bot, chatId, n)
             }
 
+            const testingDefaultPattern = /test ([a-zA-Z]+)/
+            if (testingDefaultPattern.test(userMessage)) {
+                const stateUserMessage = testingDefaultPattern.exec(userMessage)[1]
+                const testingNumberPattern = /test ([a-zA-Z]+) (\d+)/
+                let n
+                if (testingNumberPattern.test(userMessage)) {
+                    n = Number(testingNumberPattern.exec(userMessage)[2])
```

Comment:
I don't want the number functionality here because it doesn't makes sense to me to do that.

Reply:
The reason I added it is because some states have way too many testing centers and I didn't to flood the UI with many results, but sure we can remove it

Reply:
We want limited numbers of results in the result but we don't want to provide any such functionality to user.

Reply:
Sure, I'll remove it